xbmc/profiles/dialogs/GUIDialogProfileSettings.cpp

```diff
@@ -347,11 +347,11 @@ void CGUIDialogProfileSettings::InitializeSettings()
     }
 
     TranslatableIntegerSettingOptions entries;
-    entries.push_back(TranslatableIntegerSettingOption(20062, 0));
-    entries.push_back(TranslatableIntegerSettingOption(20063, 1));
-    entries.push_back(TranslatableIntegerSettingOption(20061, 2));
+    entries.emplace_back(20062, 0);
+    entries.emplace_back(20063, 1);
+    entries.emplace_back(20061, 2);
     if (profileManager->GetMasterProfile().getLockMode() != LOCK_MODE_EVERYONE)
-      entries.push_back(TranslatableIntegerSettingOption(20107, 3));
+      entries.emplace_back(20107, 3);
```

Comment:
Please revert the removal of the ctor call. It is not given that `TranslatableIntegerSettingOption` will actually stay a `std::pair` forever.